plugins/cassandra/cassandraembed.cpp

```diff
@@ -302,6 +308,7 @@ class CassandraRetryingFuture : public CInterface
             CassError rc = cass_future_error_code(future);
             if(rc == CASS_OK)
                 return true;
+            Sleep(10);
```

Comment:
It's a good guess. I will make it configurable in a future PR when I look into caching/sharing connections.